DISCOVER/_static/languages.json

```diff
@@ -0,0 +1,28 @@
+[
+    {
+        "language": "en",
+        "name": "English",
+        "preferred": true,
+        "url": "https://discover-cookbook.numfocus.org/2.0/en"
```

Comment:
I would not add the url key. I think we should only need to update the languages metadata whenever we add/remove languages or update their status